test/controllers/github_profiles_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class GithubProfilesControllerTest < ActionDispatch::IntegrationTest
```

Comment:
I don't know how test works, so I assume this is what you find in the documentation. Perhaps, it will be interesting to understand / learn how this point works @Kimon-Haars . But I approve from my side anyway now.